feature-matcher-generator-e2e-test/src/main/java/ru/yandex/qatools/proc/test/Lord.java

```diff
@@ -8,7 +9,7 @@
 
 @GenerateMatcher
 public class Lord {
-
+    @DoNotGenerateMatcher
```

Comment:
Matcher for this field won't be created. What do you think? May be we need to print message in console with warning? I'll add test for this case.